app/views/catalog/_reviewer_action.html.erb

```diff
@@ -0,0 +1,3 @@
+<% unless document.reviewed %>
+  <%= link_to "Review", reviewer_path(:id => document.id) %>
```

Comment:
Yeah, right now it will go to the reviewer path for that specific item. I've realized that you have a `review` method in the `ReviewingAsset` decorator, so I think I can use it here to mark the item as reviewed?